components/Blockquote/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,59 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+import "../index.css";
+
+<Meta
+  title="CSS Components/Data Display/Blockquote"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Blockquote" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
```

Comment:
If I check the component on MDN https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote I prefer their HTML semantic, it's more cleaner since there's no (unnecessary) `<div>` with it.

Reply:
@paulAcato This is based on the Utrecht way https://nl-design-system.github.io/utrecht/?path=/docs/css-component-blockquote--blockquote

Reply:
I agree, if there's a semantic tag we should use it